runtime/zeitgeist/src/xcm_config/fees.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 Zeitgeist PM LLC.
+//
+// This file is part of Zeitgeist.
+//
+// Zeitgeist is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at
+// your option) any later version.
+//
+// Zeitgeist is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Zeitgeist. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{xcm_config::asset_registry::CustomMetadata, Balance, CurrencyId};
+use core::marker::PhantomData;
+use frame_support::weights::constants::{ExtrinsicBaseWeight, WEIGHT_PER_SECOND};
+use xcm::latest::MultiLocation;
+use zeitgeist_primitives::constants::BalanceFractionalDecimals;
+use zrml_swaps::check_arithm_rslt::CheckArithmRslt;
```

Comment:
may be this a sign that we should move check_arith_rslt to a common location. Although not required in this PR.

Reply:
Yes I agree, we probably should make the fixed point math a standalone crate.